### ISSUE TITLE
404 not found from search result link.

### ISSUE THREAD

**Issue by appleboy** [gin-gonic/website#56]
See the result: https://gin-gonic.com/search/?q=POST

**Comment by thinkerou** [comment]
@appleboy I can search:
![image](https://user-images.githubusercontent.com/4488261/53786673-60c8fc80-3f57-11e9-9de2-f110c1a0b6a0.png)

Because we use google search, maybe you need VPN?

**Comment by appleboy** [comment]
* https://gin-gonic.com/api-example/grouping-routes/

404 not found

![search_results___gin](https://user-images.githubusercontent.com/21979/53786833-dc2aae00-3f57-11e9-9362-3d97b74c6dd4.png)

**Comment by thinkerou** [comment]
Oh, google seem to cache old url index, maybe.

**Comment by thinkerou** [comment]
@appleboy @javierprovecho I add you to Google custom search as admin, see https://cse.google.com/cse/setup/basic?cx=005538359705255204576%3Aq0elqbxmlfs

**Comment by thinkerou** [comment]
I have applied removed outdated content according https://support.google.com/websearch/answer/6349986?hl=en

**Comment by appleboy** [comment]
Still can't open the link: https://gin-gonic.com/search/?q=gin

Maybe we should revert the readme in gin repo first.

https://github.com/gin-gonic/gin/pull/1846
https://github.com/gin-gonic/gin/issues/1838
https://github.com/gin-gonic/gin/issues/1844

**Comment by thinkerou** [comment]
@appleboy OK

**Comment by appleboy** [comment]
nothing results for any search keywords.

**Comment by thinkerou** [comment]
> nothing results for any search keywords.

Yes, I attempt to fix it, but still fail, https://cse.google.com/cse/setup/basic?cx=005538359705255204576%3Aq0elqbxmlfs
